bigbluebutton-html5/imports/ui/components/video-dock/component.jsx

```diff
@@ -40,19 +41,39 @@ class VideoElement extends Component {
   }
 
   render() {
+    let cssClass;
+    if (this.props.shared || !this.props.localCamera) {
+      cssClass = styles.sharedWebcamVideoLocal;
+    }
+    else {
+      cssClass = styles.sharedWebcamVideo;
+    }
     return (
-      <video
-        id={`video-elem-${this.props.videoId}`}
-        width={320}
-        height={240}
-        autoPlay="autoPlay"
-        playsInline="playsInline"
-      />
+      <div className={styles.videoContainer + " " + cssClass} >
+        { this.props.localCamera ?
+            <video id="shareWebcam" muted={true} autoPlay={true} playsInline={true} />
+          :
+            <video id={`video-elem-${this.props.videoId}`} autoPlay={true} playsInline={true} />
+        }
+        <div className={styles.videoText}>
+          <div className={styles.userName}>{this.props.name}</div>
+          {/*<Button
+            label=""
+            className={styles.pauseButton}
+            icon={'unmute'}
+            size={'sm'}
+            circle
+            onClick={() => {}}
+          />*/}
+        </div>
+      </div>
     );
   }
```

Comment:
Please move this component into another file. Try to keep each file with a single component.